### PR TITLE
Traffic speed pred init

### PR DESCRIPTION
**Pull request by aptx1231**
Traffic speed pred的evaluator/executor/test
Model-DCRNN

### CODE REVIEW

File: trafficdl/model/loss.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import torch
+import numpy as np
+
+
+def masked_mae_loss(y_pred, y_true):
```
Comment (WenMellors) [inline trafficdl/model/loss.py:5]: 这个要不要放到 utils 文件夹下？他的源码算 loss 没有用 torch 自己提供的 Loss 函数吗

**Review by WenMellors — COMMENTED** [review]
trafficdl/model/layers.py 与 trafficdl/model/init.py 这两个空文件是什么？

**Comment by aptx1231** [comment]
> trafficdl/model/layers.py 与 trafficdl/model/init.py 这两个空文件是什么？

看的Recoble，定义公用的层和初始化函数，虽然现在没用

File: trafficdl/evaluator/traffic_speed_pred_evaluator.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import os
+import json
+import datetime
+from trafficdl.utils import ensure_dir
+from trafficdl.model import loss
+from logging import getLogger
+
+class TrafficSpeedPredEvaluator(object):
```
[the remark this answers is not in the document]
Comment (aptx1231) [reply trafficdl/evaluator/traffic_speed_pred_evaluator.py]: 已改

**Review by WenMellors — APPROVED** [review]
